qiskit/visualization/__init__.py

```diff
@@ -95,7 +235,7 @@
 
    visualize_transition
 
-Array/Matrix Visualizations
+Array/Matrix Visualizations21
```

Comment:
Something's up here.

Reply:
Not sure what happened, but I fixed it now.